tests/integ/modin/frame/test_apply.py

```diff
@@ -544,7 +544,6 @@ def test_axis_1_multi_index_column_labels_different_levels_negative():
     )
 
 
-@pytest.mark.skip(reason="SNOW-1358681")
```

Comment:
what about other skipped ones? are they not due to this?

Reply:
They have other causes. `test_axis_1_return_not_json_serializable_label` failed because the error message changed. `test_axis_1_date_time_timestamp_type` failed because snowflake started returning float64 instead of int64.